### PR TITLE
[ContactStructuralMechanicsAppplication] Full remove computing domain

### PR DESCRIPTION
**Pull request by loumalouomega**
This PR completely removes the computing domain from the ContactStructuralMechanicsAppplication

### CODE REVIEW

File: applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py [hunk 1]
```diff
@@ -74,24 +74,10 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
...
     elif solver_settings.Has("mpc_contact_settings"):  # This is a mpc contact problem
         kratos_module = "KratosMultiphysics.ContactStructuralMechanicsApplication"
         solver_module_name = "mpc_contact_" + solver_module_name
 
-        if solver_settings.Has("use_computing_model_part"):
-            if not solver_settings["use_computing_model_part"].GetBool():
-                KratosMultiphysics.Logger.PrintWarning('MechanicalSolver', 'Contact Problems must use the computing model part for now, setting it to on')
-        else:
-            solver_settings.AddEmptyValue("use_computing_model_part")
-        solver_settings["use_computing_model_part"].SetBool(True)
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py:93]: Don't you want to set it to false?

Basically what is done a few lines later for regular structural problems

**Review by philbucher — APPROVED** [review]
wow that was easier as I thought :+1: 
Thanks!

Then we can soon remove it for good :)
FYI @RiccardoRossi

File: applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py:93]: I tried, and it fails. I have an expected structure of model parts, if not followed it fails. It is everything or nothing :P

File: applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py [hunk 2]
```diff
@@ -127,6 +109,17 @@ def  AuxiliarValidateSettings(solver):
...
+    # Common settings
+    if not solver.settings["clear_storage"].GetBool():
+        KM.Logger.PrintInfo("Clear storage", "Storage must be cleared each step. Switching to True")
+        solver.settings["clear_storage"].SetBool(True)
+    if not solver.settings["reform_dofs_at_each_step"].GetBool():
+        KM.Logger.PrintInfo("Reform DoFs", "DoF must be reformed each time step. Switching to True")
+        solver.settings["reform_dofs_at_each_step"].SetBool(True)
+    if solver.settings["use_computing_model_part"].GetBool():
+        KM.Logger.PrintInfo("Using Computing-ModelPart. Switching to False")
+        solver.settings["use_computing_model_part"].SetBool(False)
```
Comment (philbucher) [inline applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py:121]: but here you do the same don't you?

Either way, you know better :)

Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py:121]: This works...(I need to fully revisit my submodelpart structure to revert those changes, and that it is going to be painful)

Comment (philbucher) [reply applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py:121]: ok

**Pull request merged**